docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -356,6 +359,79 @@ testcafe all tests/sample-fixture.js -s screenshots -p "${DATE} ${TIME}/test ${T
 
 *Overrides a configuration file property*: [screenshotPathPattern](configuration-file.md#screenshotpathpattern).
 
+### --video \<basePath\>
+
+Enables TestCafe to record videos of test runs and specifies the base directory to save these videos.
+
+```sh
+testcafe chrome test.js --video reports/screen-captures/
+```
+
+> Important! You need to install [the FFmpeg library](https://ffmpeg.org/) to record videos.
+
+If TestCafe is unable to find the FFmpeg library automatically, do one of the following:
+
+* Add the FFmpeg installation directory to the system's `PATH` environment variable;
+* Specify the path to the FFmpeg executable in the `FFMPEG_PATH` environment variable or the `ffmpegPath` parameter in [--video-options](#--video-options-optionvalueoption2value2);
+* Install the `@ffmpeg-installer/ffmpeg` package from npm.
+
+Videos are saved in the `.mp4` format.
+
+Use the [--video-options](#--video-options-optionvalueoption2value2) and [--video-encoding-options](#--video-encoding-options-optionvalueoption2value2) flags to provide options that define how videos are recorded.
+
+*Overrides a configuration file property*: [videoPath](configuration-file.md#videopath).
+
+#### Video Path Patterns
+
+The relative path to a video file and its name are composed according to the following patterns:
+
+* `${DATE}_${TIME}/test-${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.mp4` if the [quarantine mode](#-q---quarantine-mode) is disabled;
+* `${DATE}_${TIME}/test-${TEST_INDEX}/run-${QUARANTINE_ATTEMPT}/${USERAGENT}/${FILE_INDEX}.mp4` if the [quarantine mode](#-q---quarantine-mode) is enabled.
```

Comment:
We don't use a separate path pattern for quarantine mode, all quarantine attempts are recorded to a single file with the first default path pattern specified in the topic.